### ISSUE TITLE
The plugin splits the method parameters when wrapping, eclipse does not

### ISSUE THREAD

**Issue by fabien-testaniere** [revelc/formatter-maven-plugin#92]
Formatter-maven-plugin version 0.4
Eclipse Mars

<b>Scenario:</b>
Format a source file containing a method call that exceed the max line length with method calls in parameters.
Example: 
<i>myMethod(arg1, arg2, somelongStuff, var.getValue().getSomeStuff());</i>

Configure your formatter to wrap in the middle of "getSomeStuff()"

Eclipse format:
<i>myMethod(arg1, arg2, somelongStuff, 
                 var.getValue().getSomeStuff());</i>

Plugin format:
<i>myMethod(arg1, arg2, somelongStuff, var.getValue()
                 .getSomeStuff());</i>

The plugin formatter splits the argument when eclipse does not.

<b>Expecting result:</b> do not have any difference when formatting using Eclipse or using the plugin

**Comment by fabien-testaniere** [comment]
I notice the same difference between the Eclipse Kepler and Eclipse Mars formatter. Your plugin is compliant with the Eclipse Kepler format, but not with the Eclipse Mars format.

**Comment by velo** [comment]
This plugin has not been release with mars yet.

You can try my fork (that since has been merged back but not released)

He is an example!
https://github.com/revelc/formatter-maven-plugin/blob/master/pom.xml#L402

**Comment by fabien-testaniere** [comment]
Thank's for your answer. I already tried your fork before posting this defect, but I did not find how to configure a specific formatter config file. When speciying the file in the config section as described here:
http://code.revelc.net/formatter-maven-plugin/examples.html#Multimodule_Configuration
the file is found when I use the version 0.4, but is not found with the 1.6.0.RC4 version.
In your example you do not specify any config file.
